libs/SalesforceSDK/src/com/salesforce/androidsdk/push/PushNotificationsRegistrationChangeWorker.kt

```diff
@@ -67,15 +67,15 @@ internal class PushNotificationsRegistrationChangeWorker(
         } /* User account is optional where null specifies all accounts */
 
         // Instantiate push notifications registrar...
-        val pushNotificationsRegistrar = getInstance().pushServiceType.newInstance()
+        val pushNotificationsRegistrar = SalesforceSDKManager.getInstance().pushServiceType.newInstance()
```

Comment:
This isn't a result of an API change - I'll admit I changed my mind on using static imports for methods and properties with very ambiguous names like `getInstance` 😅